plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/LabelImageLoadJob.java

```diff
@@ -72,9 +57,11 @@ protected IStatus run(IProgressMonitor monitor) {
     @Override
     public void run() {
       if (!label.isDisposed()) {
-        label.addDisposeListener(new ImageDisposer(image));
-        disposerAttached = true;
-        label.setImage(image);
+        Image image = new Image(label.getDisplay(), imageData);
+        if (image != null) {
```

Comment:
I don't think this condition is possible given we invoke a constructor.

Reply:
Ah, right.